src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrderStaticGroupsBottomSeparated.java

```diff
@@ -0,0 +1,30 @@
+//non-compiled with javac: contains specially crafted set of imports for testing
+package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
+/**
+ * This test-input is intended to be checked using following configuration:
```

Comment:
I still think that location of such inputs in UTs will not prevent future regression. UT Inputs are manually crafted in weird way for testing, and on fix engineer presume that old expectations are wrong :).

One day there will be an issue, and smb who try to fix it, will make a fix and after that persuade all that all previous behavior was wrong so expected results should be different .... and we make regression. 
The same scenario was there already, that is why we missed such regression. I missed this on code review, even I try to be over caution and over demanding .... I missed it.

If we place such cases (in addition to UT) to IT, it is different story at least for me. As ITs works on strict config and changes in them is very rare. Especially if call them like "InputFromSpotify" , and name test like testSpotifyImportLayout and ... Put in Input "WARNING: do not change file as it represent exact layout in users codebase" or smth similar.

I hope this will be a red flag during code review.